src/Command/Hello.php

```diff
@@ -11,17 +11,25 @@
 
 final class Hello extends Command
 {
-    protected static $defaultName = 'hello';
-    protected static $defaultDescription = 'An example command';
```

Comment:
Should not change this since an instance of a class will be needed in order to list all commands available.